Taylor and Francis+NEJM.js

```diff
@@ -164,7 +164,30 @@ function finalizeItem(item, doc, doi, baseUrl) {
 		if (sectionheading.match(/^(Book )?Reviews?$/i))
 			item.tags.push("Book Reviews");
 	}
-
+	
+	// numbering issues with slash, e.g. in case of  double issue "1-2" > "1/2"
+	if (item.issue) item.issue = item.issue.replace('-', '/');
+	
+	//scraping orcid number 
+	let authorSectionEntries = ZU.xpath(doc, '//*[contains(@class, "contribDegrees corresponding ")]');//Z.debug(authorSectionEntries)
+	for (let authorSectionEntry of authorSectionEntries) {
+		let authorInfo = authorSectionEntry.querySelector('.entryAuthor');
+		let orcidHref = authorSectionEntry.querySelector('.orcid-author');
+		if (authorInfo && orcidHref) {
+			let author = authorInfo.childNodes[0].textContent;
+			let orcid = orcidHref.textContent.replace(/.*(\d+-\d+-\d+-\d+x?)$/i, '$1');
+			item.notes.push({note: "orcid:" + orcid + '|' + author});
+		}
+	}
+	//deduplicate
+	item.notes = Array.from(new Set(item.notes.map(JSON.stringify))).map(JSON.parse);
+	
+	//capitalize first letter
+	let tags = item.tags;
+	for (let tag in tags) {
+		item.tags.push(tags[tag].capitalizeFirstLetter());
```

Comment:
You probably duplicate the tags with this approach. Rather use a forEach-approach:
https://stackoverflow.com/questions/12482961/change-values-in-array-when-doing-foreach
But in general: We deliberated upon the question of capitalizing keywords in IxTheo some time ago but decided against it, so are a absolutely positive that this will lead to the intended result occuring cases?